L1Trigger/L1TTrackMatch/BuildFile.xml

```diff
@@ -0,0 +1,8 @@
+<use   name="DataFormats/L1Trigger"/>
```

Comment:
since build files are on my mind... this buildifle is missing a lot of direct dependencies of the package. Please add them..

Reply:
done